### DISCUSSION TITLE
Final Project Ideas

### DISCUSSION THREAD

**Discussion by CJ3315** [USU-ITLS-x265-23-summer/class-repo#60] [Ideas]
1. Portfolio- I have already coded out a portfolio in HTML but it looks like it is from the early 2000's so it would be cool to use skills from these courses and redesign/rebuild a portfolio to highlight all of my work. 
2. Blog- This would be fun to post different ideas or share things. 
3. Recruitment website- I am a recruiter for the university so it would be cool to build some sort of recruitment website so that prospective students can learn more info about USU.
4. Interactive game- This would just be a fun project that I could share with others. 
5. Learning code tutorial- This would be cool for future learners to gain insight from. 
6. Typing tutorial- This would be a helpful website for people wanting to get stronger at typing.

**Comment by metasean** [comment]
- Portfolio || Blog || Recruitment website
  - These are relatively safe, utilitarian projects.
  - You can make each of them as easy or challenging as you want.
  - You can also make each of them as vanilla or creative as you want
        e.g. https://enhancv.com/blog/the-23-most-creative-resume-designs-weve-seen/

- Interactive game
  - If you want to go this route, just keep it simple 
  - Simple online games, can be, well simple.
    - https://devdevshow.com/javascript-games-examples-for-beginners/
    - https://webtips.dev/javascript-game-ideas 
  - You'd only need to use some HTML, CSS, and JavaScript.
  - Yes, they can be time-wasting fun also. 😁 

- Learning Code tutorial
  - If you go this route, are you thinking y'all build your own, or try to add one to an existing platform?
 
- Typing tutorial
  - This seems like a somewhat specific type of project (no pun intended), but I feel like I'm lacking some context.
  - What's your motivator for a typing tutorial?
  - To be clear, no judgement. It just seems sort of random, and I'm trying to understand your reason, drive, curiosity.

**Comment by derekscholes** [comment]
These are very similar to my topics, but I feel like any would be useful. I think the first two or three can be combined if you decide to keep them simple. I decided to do this based on the similar feedback I received.
